starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -49,4 +49,27 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+  
+  /**
+   * Test Create method 
+   */
+  @Test
+  public void create() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    Inventory actualInventoryItem = this.inventoryDAO.create(inventory);
+    Assert.assertEquals(PRODUCT_TYPE, actualInventoryItem.getProductType());
```

Comment:
- Replace the constants (e.g., `PRODUCT_TYPE` and `NAME`) with the test inventory's getters since we want to directly compare the inventories

- Add an assertion to ensure `actualInventoryItem` is not null

- Add another assertion to make sure `inventory` id and `actualInventoryItem` id are not equal to each other